jenkins-x-bdd.yml

```diff
@@ -122,7 +122,7 @@ pipelineConfig:
                 image: gcr.io/jenkinsxio/builder-go:${inputs.params.version}
                 command: ./jx/scripts/ci.sh
 
-              - name: stash_xml_report
+              - name: stash-xml-report
```

Comment:
using dashes to be consistent with the other names and also because of the UI splitting on '-' (which seems odd)